comps/retrievers/langchain/pathway/requirements.txt

```diff
@@ -0,0 +1,13 @@
+docarray[full]
+fastapi
+frontend==0.0.3
+huggingface_hub
+langchain_community == 0.2.0
+langsmith
```

Comment:
I'd remove `langsmith` and add it to `requirements-runtime.txt` and run it through `entrypoint.sh` in your Docker.
Please see here and how it's done and let me know if you have any questions:

https://github.com/opea-project/GenAIComps/tree/main/comps/llms/summarization/tgi
https://github.com/opea-project/GenAIComps/blob/main/comps/llms/summarization/tgi/requirements.txt
https://github.com/opea-project/GenAIComps/blob/main/comps/llms/summarization/tgi/requirements-runtime.txt
https://github.com/opea-project/GenAIComps/blob/main/comps/llms/summarization/tgi/entrypoint.sh
https://github.com/opea-project/GenAIComps/blob/main/comps/llms/summarization/tgi/Dockerfile#L27